WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1269,6 +1270,17 @@ public boolean onPrepareOptionsMenu(Menu menu) {
             contentInfo.setVisible(false); // only show the menu item when for Gutenberg
         }
 
+        if (helpMenuItem != null) {
+            if (mEditorFragment instanceof GutenbergEditorFragment
+                && BuildConfig.DEBUG
+                && showMenuItems
+            ) {
+                helpMenuItem.setVisible(true);
+            } else {
+                helpMenuItem.setVisible(false);
+            }
```

Comment:
Just a personal preference (and not remotely close to a blocker), but I prefer to extract out variable in cases like this:

```
val helpMenuVisible = mEditorFragment instanceof GutenbergEditorFragment 
                                 && BuildConfig.DEBUG
                                 && showMenuItems
helpMenuItem.setVisible(helpMenuVisible)
```

Reply:
Thanks @mchowning ! I'll keep that in mind for my next round of PRs for this feature.